ansible/roles/docker/templates/ubi81.Dockerfile.j2

```diff
@@ -25,6 +25,7 @@ RUN dnf install --disableplugin=subscription-manager -y \
       python3 \
       openssl-devel \
       procps-ng \
+      libcap-devel \
```

Comment:
@danbev This is failing with:
```
No match for argument: libcap-devel
Error: Unable to find a match: libcap-devel
```

On one of the UBI 8 containers:
```
bash-4.4$ dnf search libcap
2021-04-12 09:39:17,189 [ERROR] dnf:22718:MainThread @logutil.py:194 - [Errno 13] Permission denied: '/var/log/rhsm/rhsm.log' - Further logging output will be written to stderr
Not root, Subscription Management repositories not updated
Extra Packages for Enterprise Linux Modular 8 - x86_64                                                                      510 kB/s | 557 kB     00:01
Extra Packages for Enterprise Linux 8 - x86_64                                                                               11 MB/s | 9.2 MB     00:00
Red Hat Universal Base Image 8 (RPMs) - BaseOS                                                                              2.7 MB/s | 778 kB     00:00
Red Hat Universal Base Image 8 (RPMs) - AppStream                                                                            18 MB/s | 5.2 MB     00:00
Red Hat Universal Base Image 8 (RPMs) - CodeReady Builder                                                                    47 kB/s |  15 kB     00:00
=============================================================== Name Exactly Matched: libcap ===============================================================
libcap.i686 : Library for getting and setting POSIX.1e capabilities
libcap.x86_64 : Library for getting and setting POSIX.1e capabilities
============================================================== Name & Summary Matched: libcap ==============================================================
libcap-ng-devel.i686 : Header files for libcap-ng library
libcap-ng-devel.x86_64 : Header files for libcap-ng library
=================================================================== Name Matched: libcap ===================================================================
libcap-ng.x86_64 : An alternate posix capabilities library
libcap-ng.i686 : An alternate posix capabilities library
bash-4.4$
```
so it looks like `libcap-devel` isn't available on UBI.

Reply:
Hmm, from what I understand it is available but only if we register using the subscription manager and then find the pool-id of the repo, and then attach to it. After that it would be available, but that feels like something that we don't want to have to do. I'm going to try to figure out if there are other options.

Reply:
Would compiling this be an option?
```console
$ dnf install -y git gcc make glibc-static
$ git clone https://github.com/mhiramat/libcap
$ cd libcap
$ make && make install
```
Which seems to compile and link properly:
```console
$ gcc -o cap -lcap -xc - <<HERE                                                      
> #include <sys/capability.h>                                                        
> #include <stdio.h>                                                                 
> int main(int argc, char** argv) {                                                  
>   printf("bajja\n");                                                               
>   return 0;                                                                        
> }                                                                                  
> HERE
$ ldd cap 
	linux-vdso.so.1 (0x00007ffff7ffa000)
	libcap.so.2 => /lib64/libcap.so.2 (0x00007ffff7bce000)
	libc.so.6 => /lib64/libc.so.6 (0x00007ffff780b000)
	/lib64/ld-linux-x86-64.so.2 (0x00007ffff7dd4000)

```

Reply:
@danbev We can give it a try 🤞